components/o-expander/stories/nested-expander.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import {Expander} from '../src/tsx/expander';
+import javascript from '../main.js';
+import './expander.scss';
+import {useEffect} from 'react';
+
+export const NestedExpander = ({children, ...args}) => {
+	useEffect(() => void javascript.init(), []);
+	return (
+		<Expander {...args} header={<h3>First Expander</h3>}>
+			<p>Details of first expander</p>
+			<ul>
+				<li>one</li>
+				<li>two</li>
+				<li>three</li>
+				<li>four</li>
+			</ul>
+			<button className="o-expander__toggle">Secondary toggler button first expander</button>
+			<div className="innerExpanderItem">
+				<Expander {...args} header={<h3>Second Expander</h3>}>
+				<p>Second expander details</p>
+				<ul>
+					<li>ten</li>
+					<li>eleven</li>
+					<li>twelve</li>
+					<li>thirteen</li>
+				</ul>
+			</Expander>
+			</div>
+		</Expander>
+	);
+};
+
+export default {
+	title: 'Components/o-expander/nested',
```

Comment:
```suggestion
	title: 'Components/o-expander',
```

I don't think we need to group by `nested` here
![Screenshot 2024-04-03 at 16 14 10](https://github.com/Financial-Times/origami/assets/10405691/27b70c3c-9863-45f4-9378-a5c05429afb4)